lib/tasks/migrate_site_banners.rake

```diff
@@ -0,0 +1,18 @@
+namespace :site_banners do
+  task :migrate_site_banners => :environment do
+    # Update record_id's to use uuids instead of bigints
+    SiteBanner.all.each do |site|
+      attachment = ActiveStorage::Attachment.find(legacy_record_id: site.legacy_id)
+      attachment.record_id = site.id
+      attachment.update
+    end
+
+    # Set existing banners to default
+    Publisher.all.each do |publisher|
```

Comment:
let's use `find_each` for collections > 1000 https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each

Reply:
Done ✔️ I'm thinking using `find_each` is enough optimization here, given we'll only run this task once during a maintenance mode. Unless you think we'll run into problems iterating over Publishers, lmk your thoughts.